docs/contributing/introduction-to-contributing.md

```diff
@@ -16,66 +16,29 @@ please read the [code of conduct](./code-of-conduct.md) and follow the direction
-3. Check the project's README to determine how to run the project locally.
-
-For many of our repositories, you'll find the directions below work for running the project locally:
-
-Run `npm ci` to install the dependencies and set up the project.
-
-You can also use the shell commands below to get started once you have forked the repository:
-
-```shell
-git clone https://github.com/<GITHUB_USERNAME>/<REPO_NAME>.git
-cd <REPO_NAME>
-npm ci
-```
-
-Make sure to replace `<GITHUB_USERNAME>` with your GitHub username.
-
-## Building
-
-To generate a production-ready version of your code, run:
-
-```shell
-npm run build
-```
-
-## Testing
+## Issues
 
-For running the test suite, use the following command:
+If you wish to work on an open issue, please comment on the issue with `.take`, and it will be assigned to you. If an issue is not assigned, it is assumed to be available for anyone to work on. Please assign yourself to an issue before working on it to avoid conflicts.
 
-```shell
-# the tests will run in watch mode by default
-npm test
-```
+If you contribute to the project for the first time, you can search for issues with `good first issue` or `bug` labels in the repository.
 
-Since the tests run in watch mode by default, some users may encounter errors about too many files being open. In this case, it may be beneficial to [install Watchman](https://facebook.github.io/watchman/docs/install.html).
+In case you get stuck, feel free to ask for help in the [Discord](https://discord.gg/U2peSNf23P) server or [GitHub Discussions](https://github.com/orgs/open-sauced/discussions).
 
-Check out this course, [Testing JavaScript](https://testingjavascript.com/), for more info on testing React and JavaScript.
+Please follow our [code of conduct](./code-of-conduct.md) in all your interactions with the project and its contributors.
 
-## Applying Lint Styleguide
+## Commits
 
-To check the code for formatting and linting errors, run the following command:
+When contributing to OpenSauced projects, you can use this command to follow our conventional commits:
 
 ```shell
-npm run lint
+npm run push
 ```
 
-These errors will also be displayed during development but won't prevent the code from compiling.
-
-To fix the formatting and linting errors, run the following command instead:
-
-```shell
-npm run format
-```
+Using the `npm run push` command is an interactive replacement for `git commit`. It enforces the conventional commit specification for writing commit messages, making it easier for developers and maintainers to understand the changes made in a particular commit.
 
-These commands use [ESLint](https://eslint.org/) to check and fix the code.
+Assuming you are dealing with code changes and you add them using `git add`, once you are ready to commit, there are two ways we can proceed: `git commit` or `npm run push`. The second method is preferred, as doing a subsequent `git push` and opening a PR would ensure the title conforms to our standards.
```

Comment:
I'm thinking to move this paragraph up as the intro paragraph for the Commits section and adjust the wordings a little bit. Any thoughts?

```suggestion
Assuming you are dealing with code changes and add them using `git add`. Once you are ready to commit, there are two ways you can proceed: `git commit` or `npm run push`. At OpenSauced, we prefer the second method, as doing a subsequent `git push` and opening a PR would ensure the title conforms to our standards.
```

Reply:
Since it seems like there is a heavy preference to use `npm run push` it might be better to only present that option instead of saying "you can use git commit or npm run push"

The way I see it, new contributors to open source will probably just default to whatever the preferences that are laid out in the directions.
More regular contributors to open source will either default to the preference or just default to their preference of using git commit anyway.

Reply:
That's make sense. Thanks! 👍